### ISSUE TITLE
Import errors

### ISSUE THREAD

**Issue by abunch** [mautic/mautic#11080]
### Mautic Version

4.2.x series

### PHP version

7.4.3

### What browsers are you seeing the problem on?

Not relevant

### What happened?

`[2022-04-17 17:47:04] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "strpos() expects parameter 1 to be string, int given" at /var/www/html/app/bundles/LeadBundle/EventListener/ImportContactSubscriber.php line 198 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): strpos() expects parameter 1 to be string, int given at /var/www/html/app/bundles/LeadBundle/EventListener/ImportContactSubscriber.php:198)"} []`

Receiving import errors that weren't there prior to this PR:  https://github.com/mautic/mautic/pull/10884

### How can we reproduce this issue?

Create a simple .csv file with a single header line (firstname, lastname, email) and a single contact to import (john, smith, test@test.com) import from csv and select import in background.  I've been able to (after feeding mautic the same file multiple times) get the file to successfully import with no changes so this seems like a race condition.

### Relevant log output

```shell
`[2022-04-17 17:47:04] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "strpos() expects parameter 1 to be string, int given" at /var/www/html/app/bundles/LeadBundle/EventListener/ImportContactSubscriber.php line 198 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): strpos() expects parameter 1 to be string, int given at /var/www/html/app/bundles/LeadBundle/EventListener/ImportContactSubscriber.php:198)"} []`
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by jos0405** [comment]
Hi, trying but can't reprodice it. Import works fine with the file you mentioned. Also the fact, that "sometimes works" points me at an environment issue.
Are you trying to import custom fields as well?

**Comment by RCheesley** [comment]
Hi @abunch could you provide some feedback as above?  Otherwise we will have to close the issue. Thanks in advance!

**Comment by abunch** [comment]
https://user-images.githubusercontent.com/2077840/168889009-0f3c7a87-0b0e-4269-b527-05aaec957f7e.mov

**Comment by abunch** [comment]
You can close it if you want, but it's definitely broken.

**Comment by haneefputtur** [comment]
Hi,

I am facing exact same issue while importing. it worked for 2-3 times intermittently .. now again same issue i am facing.


[2022-05-31 06:10:20] mautic.WARNING: File /home/ddd/public_html/mailing/app/../var/tmp/imports/20220531060728.csv was removed. [] {"hostname":"server.suddi.org","pid":16162}
[2022-05-31 06:10:20] mautic.WARNING: Import for file /home/ddd/public_html/mailing/app/../var/tmp/imports/20220531060728.csv was aborted because the form is empty. [] {"hostname":"server.dddddddd.org","pid":16162}
[2022-05-31 06:11:34] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "strpos() expects parameter 1 to be string, int given" at /home/esgrs/public_html/mailing/app/bundles/LeadBundle/EventListener/ImportContactSubscriber.php line 198 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): strpos() expects parameter 1 to be string, int given at /home/esgrs/public_html/mailing/app/bundles/LeadBundle/EventListener/ImportContactSubscriber.php:198)"} {"hostname":"server.dddd.org","pid":16497}

**Comment by bartoszpijet** [comment]
Seems like issue only appeares while "Don't override value...." is enabled, at least in my case

**Comment by abunch** [comment]
> Seems like issue only appeares while "Don't override value...." is enabled, at least in my case

Same

**Comment by vicampuzano** [comment]
> Seems like issue only appeares while "Don't override value...." is enabled, at least in my case

Same

**Comment by dlackovic2** [comment]
> Seems like issue only appeares while "Don't override value...." is enabled, at least in my case

Same case here. When this option is disabled, it is working just fine.

**Comment by dipesh008** [comment]
Same issue, import fails 
1. i have custom fields
2. I have "Dont Override value set to YES"

**Comment by davext** [comment]
+1 here

**Comment by stevedrobinson** [comment]
Same here -> only when do not override value is "YES"

**Comment by toondeg** [comment]
+ 1

**Comment by RCheesley** [comment]
Please test #11350 which should fix this issue - thanks for reporting it folks! We need two tests to get the fix merged, testing instructions can be found here: https://contribute.mautic.org/contributing-to-mautic/tester

**Comment by escopecz** [comment]
Closing as the PR was merged and will be released in 4.4.1
